src/dctest/outcome.clj

```diff
@@ -17,6 +21,19 @@
                                 (dctest.outcome/fail! ~sts
                                                       {:message (str "Exception thrown: " (.-message ~err))}))))
                    forms)]
+    ;; Repeatedly (re)define `sts` gensym with each transformed form above.
+    ;; Conceptually expanding:
+    ;;
+    ;; (pending-> x f g h)
+    ;;
+    ;; to:
+    ;;
+    ;; (P/let [sts x
+    ;;         sts (f' sts)
+    ;;         sts (g' sts)]
+    ;;   (h' sts))
```

Comment:
Why the ticks in the conceptual transformation?

Reply:
f/g/h are the original forms, and we are transforming substantially by wrapping in try/catch and pending? checks (in addition to the normal `->` transformation).  It felt like a lie to just write f/g/h, so the prime was attempting to indicate that transformation ... maybe it confuses more than it helps?